sdk/cognitiveservices/pom.service.xml

```diff
@@ -9,23 +9,29 @@
   <packaging>pom</packaging>
   <version>1.0.0</version><!-- Need not change for every release-->
   <modules>
-    <module>ms-azure-cs-autosuggest</module>
-    <module>ms-azure-cs-computervision</module>
-    <module>ms-azure-cs-contentmoderator</module>
-    <module>ms-azure-cs-customimagesearch</module>
-    <module>ms-azure-cs-customsearch</module>
-    <module>ms-azure-cs-customvision-prediction</module>
-    <module>ms-azure-cs-customvision-training</module>
-    <module>ms-azure-cs-entitysearch</module>
-    <module>ms-azure-cs-faceapi</module>
-    <module>ms-azure-cs-imagesearch</module>
-    <module>ms-azure-cs-luis-authoring</module>
-    <module>ms-azure-cs-luis-runtime</module>
-    <module>ms-azure-cs-newssearch</module>
-    <module>ms-azure-cs-spellcheck</module>
-    <module>ms-azure-cs-textanalytics</module>
-    <module>ms-azure-cs-videosearch</module>
-    <module>ms-azure-cs-visualsearch</module>
-    <module>ms-azure-cs-websearch</module>
+    <module>../core/azure-core</module>
```

Comment:
should we revert this file changes now that we don't have it under cognitive services?